src/main/java/at/ac/fhcampuswien/fhmdb/controller/WatchlistController.java

```diff
@@ -0,0 +1,65 @@
+package at.ac.fhcampuswien.fhmdb.controller;
+
+import at.ac.fhcampuswien.fhmdb.FhmdbApplication;
+import at.ac.fhcampuswien.fhmdb.exceptions.DatabaseException;
+import at.ac.fhcampuswien.fhmdb.models.Movie;
+import at.ac.fhcampuswien.fhmdb.models.WatchlistEntity;
+import com.jfoenix.controls.JFXButton;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.layout.StackPane;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WatchlistController extends AbstractViewController {
+
+    @FXML
+    private StackPane parent;
+    @FXML
+    protected JFXButton homeButton;
+
+
+    @Override
+    public void initialize() {
+        super.initialize();
```

Comment:
### Remove Button not shown in the Movie-Cell

Add this before super.initialize():
`this.isWatchlistCell = true;`